### PR TITLE
single-file 2.0.29 (new formula)

### PR DESCRIPTION
**Pull request by AngelaDMerkel**
Add single-file 2.0.29

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by AngelaDMerkel** [comment]
Performing the tests on my system does not produce any errors whatsoever, so I can't understand precisely what the issue is here.

File: Formula/s/single-file.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system "false"
```
Comment (chenrui333) [inline Formula/s/single-file.rb:18]: this is wrong, check other nodejs formula tests.

File: Formula/s/single-file.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class SingleFile < Formula
+  desc "CLI tool for saving a faithful copy of a complete web page in a single HTML file"
+  homepage "https://github.com/gildas-lormeau/single-file-cli/blob/master/README.MD"
```
Comment (chenrui333) [inline Formula/s/single-file.rb:5]:
```suggestion
  homepage "https://github.com/gildas-lormeau/single-file-cli/"
```

**Comment by AngelaDMerkel** [comment]
This should be resolved now, but why does updating the pull request break the commit style guide?

**Pull request closed without merging**